src/test/java/com/zollingpaper/backend/paper/controller/PaperControllerE2ETest.java

```diff
@@ -0,0 +1,102 @@
+package com.zollingpaper.backend.paper.controller;
+
+import static org.hamcrest.Matchers.is;
+import static org.junit.jupiter.api.DynamicTest.dynamicTest;
+
+import com.zollingpaper.backend.board.domain.BoardFixture;
+import com.zollingpaper.backend.global.DatabaseCleaner;
+import com.zollingpaper.backend.paper.domain.PaperFixture;
+import com.zollingpaper.backend.paper.exception.PaperErrorCode;
+import io.restassured.RestAssured;
+import io.restassured.http.ContentType;
+import java.util.stream.Stream;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.DynamicTest;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.context.SpringBootTest.WebEnvironment;
+import org.springframework.boot.test.web.server.LocalServerPort;
+
+@SpringBootTest(webEnvironment = WebEnvironment.RANDOM_PORT)
+public class PaperControllerE2ETest {
+
+    @LocalServerPort
+    int serverPort;
```

Comment:
```suggestion
private int serverPort;
```

접근제어자 private으로 해도 괜찮을 것 같습니다!